src/main/java/baritone/cache/CachedRegion.java

```diff
@@ -43,7 +43,7 @@
     /**
      * Magic value to detect invalid cache files, or incompatible cache files saved in an old version of Baritone
      */
-    private static final int CACHED_REGION_MAGIC = 456022910;
+    private static final int CACHED_REGION_MAGIC = 456022911;
```

Comment:
am I supposed to just bump this, or roll some dice and pick new numbers, or what